lib/routes/lnurl/lnurl_invoice_delegate.dart

```diff
@@ -84,18 +94,17 @@ Future handleWithdrawRequest(
     defaultDescription: withdrawParams.defaultDescription,
   );
 
-  LNURLWithdrawPageResult? pageResult;
-  pageResult = await showDialog<LNURLWithdrawPageResult>(
-    useRootNavigator: false,
-    context: context,
-    barrierDismissible: false,
-    builder: (_) => LNURLWithdrawDialog(
-        requestData: requestData, domain: withdrawParams.domain),
-  );
-  if (pageResult == null) {
-    return Future.value();
-  }
-  if (pageResult.error != null) {
-    throw pageResult.error.toString();
-  }
+  Completer<LNURLWithdrawPageResult?> completer = Completer();
+  Navigator.of(context).push(MaterialPageRoute(
+    builder: (_) => CreateInvoicePage(
+      requestData: requestData,
+      domain: withdrawParams.domain,
+      onFinish: (LNURLWithdrawPageResult? response) {
```

Comment:
@ademar111190  Is onFinish called also when the user canceled the invoice creation (e.g go back)?
Asking because I am not sure how this affect the caller of this function that waits with a promise that won't be resolved.

Reply:
Good question; it could lead to a leak. I'm adding a check on the disposal moment to call the finish function passing null if it was not called yet so we avoid it.

Reply:
I don't see this fix. Can you refer me to this change?